velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -496,21 +502,25 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
 core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
     const ::substrait::Plan& substraitPlan,
     memory::MemoryPool* pool) {
+  VELOX_CHECK(
+      checkTypeExtension(substraitPlan),
```

Comment:
The details about the UNKNOWN type are hard-coded in multiple places. It would be nice to refactor to consolidate these. This can be done in a follow-up PR though.

Specifically, checkTypeExtension could be modified to throw so that error message can be moved next to the check.

Reply:
sure.